packages/account/src/predicate/predicate.ts

```diff
@@ -28,6 +28,14 @@ import type {
 
 import { getPredicateRoot } from './utils';
 
+type PredicateConstructorArgs<T> = {
```

Comment:
If I'm not mistaken, we use the suffix params in these cases, no?

I am unsure; feel free to dismiss if that's not the case.

Here's a suggestion:

```suggestion
type PredicateParams<T> = {
```